ChangeLog

```diff
@@ -1,3 +1,6 @@
+Version 5.2.1
```

Comment:
considering pom.xml is 5.2.1-SNAPSHOT, we should probably use a different version here.